src/app/components/forms/user/password-reset/password-reset.component.html

```diff
@@ -0,0 +1,68 @@
+<h1>
+  {{
+    i18nService.i18nComponentsEnum.RESET_PASSWORD_COMPONENT_TITLE | translate
+  }}
+</h1>
+<h2>
+  {{
+    i18nService.i18nComponentsEnum.RESET_PASSWORD_COMPONENT_SUBTITLE | translate
+  }}
+</h2>
+
+<form
+  [formGroup]="passwordResetForm"
+  (ngSubmit)="onSubmit()"
+  autocomplete="off"
+  class="password-reset-form"
+>
+  <mat-form-field appearance="outline" class="full-width">
```

Comment:
misma historia con el class="full-width"


```suggestion
  <mat-form-field appearance="outline" class="w-100">
```